client/galaxy/scripts/mvc/workflow/workflow-manager.js

```diff
@@ -2,6 +2,25 @@ import $ from "jquery";
 import Connector from "mvc/workflow/workflow-connector";
 import { Toast } from "ui/toast";
 
+import Vue from "vue";
+import MarkdownEditor from "components/Markdown/MarkdownEditor";
+
+const DEFAULT_INVOCATION_REPORT = `
+# Workflow Execution Report
+
+## Workflow Inputs
+::: invocation_inputs
```

Comment:
That needs to be updated to the new syntax.

Reply:
Done - good catch thanks!